src/Elcodi/Admin/BannerBundle/DependencyInjection/AdminBannerExtension.php

```diff
@@ -37,7 +37,7 @@ class AdminBannerExtension extends AbstractExtension
      */
     public function getConfigFilesLocation()
     {
-        return __DIR__ . '/../Resources/config';
+        return __DIR__.'/../Resources/config';
```

Comment:
This are the kind of changes that fabpot's php-fix does but we don't want to have at Elcodi coding style.
/ping @mmoreram